tools/distribution/src/release/assets/gh_asset.py

```diff
@@ -45,6 +45,19 @@ def validate(self, zip_directory: DirectoryMatcher):
             'ios-arm64_x86_64-simulator/**/arm64-apple-ios-simulator.swiftinterface',
             'ios-arm64_x86_64-simulator/**/x86_64.swiftinterface',
             'ios-arm64_x86_64-simulator/**/x86_64-apple-ios-simulator.swiftinterface',
+
+            'tvos-arm64',
+            'tvos-arm64/BCSymbolMaps/*.bcsymbolmap',
+            'tvos-arm64/dSYMs/*.dSYM',
+            'tvos-arm64/**/arm64.swiftinterface',
+            'tvos-arm64/**/arm64-apple-ios.swiftinterface',
+
+            'tvos-arm64_x86_64-simulator',
+            'tvos-arm64_x86_64-simulator/dSYMs/*.dSYM',
+            'tvos-arm64_x86_64-simulator/**/arm64.swiftinterface',
+            'tvos-arm64_x86_64-simulator/**/arm64-apple-ios-simulator.swiftinterface',
+            'tvos-arm64_x86_64-simulator/**/x86_64.swiftinterface',
+            'tvos-arm64_x86_64-simulator/**/x86_64-apple-ios-simulator.swiftinterface',
```

Comment:
These slices won't be available when running against `1.9.0` and earlier. I think we need to leverage / extend the logic around `should_be_included` somehow:
```python
def should_be_included(self, in_version: Version):
    return True  # always expect `Datadog.xcframework`
```